mkdumprd

```diff
@@ -60,9 +60,12 @@ add_dracut_sshkey()
 # caller should ensure $1 is valid and mounted in 1st kernel
 to_mount()
 {
-	local _target=$1 _fstype=$2 _options=$3 _sed_cmd _new_mntpoint _pdev
+	local _target=$1 _fstype=$2 _options=$3 _new_mntpoint=$4
+	local _sed_cmd _pdev
 
-	_new_mntpoint=$(get_kdump_mntpoint_from_target "$_target")
+	if [[ -z "$_new_mntpoint" ]]; then
+		_new_mntpoint=$(get_kdump_mntpoint_from_target "$_target")
+	fi
```

Comment:
```
_new_mntpoint="${_new_mntpoint:-$(get_kdump_mntpoint_from_target "$_target")}"
```
is identical but shorter and better matches what is done for the other options right below.

Reply:
Agreed